src/JustSaying/AwsTools/MessageHandling/IamSqsPolicyBuilder.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Amazon;
+
+namespace JustSaying.AwsTools.MessageHandling;
+
+internal static class IamSqsPolicyBuilder{
+
+    public static string BuildPolicyJson(SqsPolicyDetails policyDetails)
```

Comment:
It feels like this could be an extension method on `SqsPolicyDetails` (and same for `SnsPolicyDetails`) over a builder. It would make the usage slightly less verbose

Reply:
Yep - easy change

Reply:
@adammorr Have switched to using extension methods for iam policy generation.